src/GslibIO.jl

```diff
@@ -13,6 +13,14 @@ using GeoStatsBase
 # type aliases
 const Array2or3{T} = Union{AbstractArray{T,2},AbstractArray{T,3}}
 
+# structure for GSLIB metadata
+struct GSLIBMetadata
+  header    # Content of the first line (documentation only)
+  nvars     # An integer with the number of variables (second line)
+  varnames  # The variable names in the following `nvars` lines
+  extrainfo # This corresponds to the extra info in the second line after `nvars`
```

Comment:
There is not wrong/right order as the structure is high level. I placed it at the end because it is optional and not actually used. I think it is much better to keep it at the end.

Reply:
> So this extrainfo contains what exactly? Since it is coming immediately after the `nvars`, I would place it in the right order in the struct.

I think the documentation is very clear. I will give you an example if you are not familirised with GSLIB format.
If you krige a regular grid using kt3d, the output will contain two variables (estimation and variance), but some versions of GSLIB add after the second line meant for the number of variables (2 is this case) the grid specification (usually just number of nodes, but sometimes nodes, starts and sizes). This extra information is just for annotate the grid specification, but is never used in any GSLIB program. Therefore, you will see something like:
```
KT3D ESTIMATES WITH:  samples_au                                                
    2   10   20   10
Estimate
EstimationVariance
0.62118578     0.81496090    
```
In this example, nvars is 2, whereas `extrainfo` will be "10   20   10".

`extrainfo` is not used, even we could not include it in the structure, but I think it is better to parse everything in case users want to do something with it.

I hope this explanation helps you to better understand the GSLIB format. You could read the official doc here: http://www.gslib.com/gslib_help/format.html.

Reply:
Based on your explanation, the `extrainfo` field should be an *arbitrary* string that is ignored in most applications. I agree we should remove it from the struct. If someone complains about it or has a use case in the future that requires parsing this extra info, we can always submit a new PR to fix the issue. Other lines are also allowed to have comments like the lines with variable names, but we don't want to save comments in general.